lib/subprocess.js

```diff
@@ -176,7 +176,7 @@ class SubProcess extends EventEmitter {
 
   async stop (signal = 'SIGTERM', timeout = 10000) {
     if (!this.isRunning) {
-      throw new Error(`Can't stop process; it's not currently running (cmd: '${this.cmd}')`);
+      throw new Error(`Can't stop process; it's not currently running (cmd: '${this.cmd} ${this.args.join(' ')}')`);
```

Comment:
the output might be confusing if args contain spaces. I'd rather use the function, which properly quotes each arg

Reply:
Sure.